README.md

```diff
@@ -85,15 +85,15 @@ If you include the same keyword in subsequent calls,
 preferring the value(s) in the API call, so with
 
 ```python
-api = BasicAPI('example.com', headers={'User-Agent': 'fancy'})
+api = BasicAPI('https://api.example.com', headers={'User-Agent': 'fancy'})
 api.get('/cool/path', headers={'User-Agent': 'super fancy'})
 ```
 
 the `get` call will override the previous `User-Agent` header,
 resulting in a "super fancy" user agent, and
 
 ```python
-api.get('/cool/path', headers={'another', 'header'})
+api.get('/cool/path', headers={'another': 'header'})
```

Comment:
This is the bug that my funky doctest caught.

Reply:
I figured 😝